templates/header.mustache

```diff
@@ -0,0 +1,4 @@
+<header>
+  <img id="logo">
+  <div id="avatar-or-signin-button"></div>
+</header>
```

Comment:
Same comment as https://github.com/mozilla/idea-town-app/pull/11/files#r38004578

This looks the same as src/index.html#L11-14.